core/domain/email_services.py

```diff
@@ -69,8 +77,9 @@ def _is_sender_email_valid(sender_email):
 
 
 def send_mail(
-        sender_email, recipient_email, subject, plaintext_body,
-        html_body, bcc_admin=False):
+        sender_email: str, recipient_email: str, subject: str,
+        plaintext_body: str, html_body: str, bcc_admin: Optional[bool] = False
+        ) -> None:
```

Comment:
```suggestion
    sender_email: str, 
    recipient_email: str, 
    subject: str,
    plaintext_body: str, 
    html_body: str, 
    bcc_admin: Optional[bool] = False
) -> None:
```

Also, why is `bcc_admin` optional?

Reply:
Because it had `False` as default value

Reply:
So? `False` is of `bool` type.

Reply:
Okay